### DISCUSSION TITLE
Problem in Projects(Beta)

### DISCUSSION THREAD

**Discussion by Alanimdeo** [community/community#19039] [Projects and Issues] [answered]
![problem](https://user-images.githubusercontent.com/32806763/174813905-69310c74-29d6-49c2-bdbc-865de7e21f65.gif)

When I click 'Open' or 'Closed' in Projects(Beta), it goes to Projects (classic).

![image](https://user-images.githubusercontent.com/32806763/174814875-1b48ee2e-7fb1-4b5e-b0b8-bd0892def6d2.png)
There are multiple type parameters, so latter should be deleted.

**Comment by melvincwng** [comment, marked as answer]
Hi @alanimdeo, for bugs, please kindly raise a ticket and report it to GitHub Support over [here](https://support.github.com/contact/bug-report)

The support engineers usually respond pretty quickly, within a couple of days, after they've done their investigation. 

If this reply has helped you, please kindly 'Mark as answer' so that the others are aware this issue has been addressed.

Thank you, and have a great day ahead 😊
